### DISCUSSION TITLE
hi why my issue got removed?

### DISCUSSION THREAD

**Discussion by Adamkaram** [medusajs/medusa#4371] [Q&A] [answered]
i had an issue and declared it in isuues section as a bug 
my main issue was 
slowly navigate to produts like 2 seconde to navigte 
and carts section doesn't work well!
 
4 request has been made for navigation to a product and that's a lot 
![Screenshot 2023-06-18 115002](https://github.com/medusajs/medusa/assets/52092726/3fbfeec7-8d6d-425a-af99-701a4898578b)

cart isn't working probably
![Screenshot 2023-06-18 115752](https://github.com/medusajs/medusa/assets/52092726/588e9faa-0918-49dd-8e09-423d8448584f)

401 status code for every navigate sometimes happens and sometimes not !
![Screenshot 2023-06-18 115849](https://github.com/medusajs/medusa/assets/52092726/9bfa323b-2bf3-49ef-9e53-ab709a94ab4a)

i'm working on 
windows 10 
latest node version 
latest project version

**Comment by RegisWeduLabs** [comment]
Just a guess, but probably because this should go in https://github.com/medusajs/nextjs-starter-medusa - or whatever front end starters you used?  There are also MANY different reasons why this would happen, your setup, environment, ressources, so it's quite hard to pinpoint...

**Comment by Adamkaram** [comment]
i used nextjs , should i delete it and create a new setup

**Reply by RegisWeduLabs** [reply, marked as answer]
You might want to create the issue here:  https://github.com/medusajs/nextjs-starter-medusa

**Reply by Adamkaram** [reply]
done

**Comment by Adamkaram** [comment]
they deleted it again there !

**Reply by RegisWeduLabs** [reply]
No they closed it and replied to you with useful information...
